mne/channels/montage.py

```diff
@@ -206,6 +206,16 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         ch_names_ = []
         pos = []
         with open(fname) as fid:
+            # Default units are meters
+            scale_factor = 1.
+            for line in fid:
+                if 'UnitPosition' in line:
+                    units = line.split()[1]
+                    if units == 'm':
+                        break
+                    elif units == 'mm':
+                        scale_factor = 1. / 1000.
+                        break
```

Comment:
Sorry not part of this clause, I mean at the end of the loop, i.e. as part of the `for`:
```
for ...
    if ...
    elif ....
else:
    raise RuntimeError(...)
```